### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hibeekaey**
This PR fixes the markdown (removes) in the README.

## Checklist

- [ ] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by raymondfeng** [comment]
@hibeekaey Thank you for the PR. But what's the reason behind the proposed change? The `README.md` is automatically formatted by `prettier` and the new one breaks it.

**Comment by hibeekaey** [comment]
Screen Shot 2020-09-25 at 11 05 22 AM

Notice the markdown still in the README?

File: examples/lb3-application/README.md [hunk 1]
```diff
@@ -95,10 +95,10 @@ applications can use them.
...
 Start the new LB4 application
 
-    ```sh
-    $ npm start
-    Server is running at http://127.0.0.1:3000
-    ```
+   ```sh
+   $ npm start
+   Server is running at http://127.0.0.1:3000
+   ```
```
Comment (achrinza) [inline examples/lb3-application/README.md:101]: Why not remove the indentation altogether?

````suggestion
```sh
$ npm start
Server is running at http://127.0.0.1:3000
```
````

**Comment by raymondfeng** [comment]
@hibeekaey I see. Good catch.

There are a few logistic steps to be done:

1. When you submit the PR, please leave the checklist and mark items if applicable
2. Please sign the commit - https://loopback.io/doc/en/contrib/code-contrib.html
3. Please run `npm run prettier:fix` to ensure code is formatted correctly

**Comment by raymondfeng** [comment]
FYI: If you run `npm ci` for `loopback-next`, it will set up git hooks for certain steps too.

**Comment by raymondfeng** [comment]
@hibeekaey Do you have a chance to address our comments?

**Comment by hibeekaey** [comment]
> @hibeekaey Do you have a chance to address our comments?

I'm currently doing that.

File: bodyparsers/rest-msgpack/tsconfig.json [hunk 2]
```diff
@@ -13,9 +13,6 @@
     {
       "path": "../../packages/core/tsconfig.json"
     },
-    {
```
Comment (dhmlau) [inline bodyparsers/rest-msgpack/tsconfig.json:16]: Could you please revert this change? It's not related to this PR. Thanks.

**Comment by dhmlau** [comment]
@hibeekaey, CI failed because of the commit linter error. Since there's one file change in this PR, please squash the commit and update the commit message to adhere our [commit message guideline](https://loopback.io/doc/en/lb4/code-contrib-lb4.html#commit-message-guidelines), for example:
```
fix(docs): update examples/lb3-application/README.md
```

**Comment by bajtos** [comment]
To speed up the process, removed the changes in `bodyparsers/rest-msgpack/tsconfig.json` and cleaned up commit history. Let's wait for CI results before landing.

**Comment by bajtos** [comment]
@hibeekaey I noticed you are using two email addressed in your commits: `ibukundairo@hackerbay.io` and `hibeekaey@yahoo.com`. Our DCO process requires the commit to be signed by the author's email.

Can you please do the following steps?

1. Run `git fetch origin` && `git reset --hard origin/patch-1` to pick up the changes I made in your feature branch.
2. Run `git commit --amend` and add (or edit) `Signed-off-by:` line to ensure the commit is signed using the author's email (`Ibukun Dairo <hibeekaey@yahoo.com>`)
3. Run `git push` to push the changes to GitHub .
